### PR TITLE
v1.4: Select account command and UG/DG

### LINKED ISSUE #47: Add adoc for Git
Let us assume that the developers reading our DG might not know how to use Git (he might be the type who transfer files via email, or uses Mercurial...who knows, right?). 

Lets update our DG to contain a sub-section on how to use Git with SourceTree.

### LINKED ISSUE #190: Add "Name" to user account
Add one more field called `Name` to the user account.

### LINKED ISSUE #264: v1.4: select-account command
`select-account INDEX`

### LINKED ISSUE #269: Wrong XML format triggers building a new but empty XML file
**Describe the bug**
If there's a data in the XML that has an incorrect tag, it triggers a new XML to be generated **but** without the `root` account.

**To Reproduce**
1. Populate XML with some data.
2. Remove some field or make some field invalid.
3. Run the app, there should be no data at all and trying to login to `root` will fail since it doesn't exists.

### PR DESCRIPTION
**Pull request by AZhiKai**
Closes #264 
Closes #190 
Closes #47 
Fixes #269 
Includes update to UGDG and my PPP.

**Ready for review**

### CODE REVIEW

File: src/main/java/seedu/restaurant/ui/MainWindow.java [hunk 1]
```diff
@@ -425,6 +430,7 @@ private void handleDisplaySalesChartEvent(DisplaySalesChartEvent event) {
         salesChartWindow.show();
     }
 
+    @Subscribe
```
Comment (AZhiKai) [inline src/main/java/seedu/restaurant/ui/MainWindow.java]: Not sure what happened but this `@Subscribe` got removed, added it back for you.

**Comment by AZhiKai** [comment]
@hyperionnkj I'm not intending to use the stack panel to display anything so I thought I'd just display both username and the name.

Would it be better if i swap the placement?

**Comment by HyperionNKJ** [comment]
I think it would be better to display the name instead of the username.

**Comment by AZhiKai** [comment]
> I think it would be better to display the name instead of the username.

Intention is to only allow the admin aka owner to view but that's for 2.0, so he should at the very least be able to see the employee's full name and the username (also select `select` and `find` works by looking at username, not full name. The only changes that I can think of doing is either letting it remain or swap the display label

**Pull request merged**